capellambse_context_diagrams/context.py

```diff
@@ -288,6 +290,7 @@ def __init__(
             "display_port_labels": False,
             "port_label_position": _elkjs.PORT_LABEL_POSITION.OUTSIDE.name,
             "transparent_background": False,
+            "display_unused_ports": False,
```

Comment:
Keep the `transparent_background` flag at the end and the port flags together.
```suggestion
            "display_port_labels": False,
            "port_label_position": _elkjs.PORT_LABEL_POSITION.OUTSIDE.name,
            "display_unused_ports": False,
            "transparent_background": False,
```